perfdash/google-gcs-downloader.go

```diff
@@ -37,6 +37,7 @@ import (
 // GoogleGCSDownloaderOptions is an options for GoogleGCSDownloader.
 type GoogleGCSDownloaderOptions struct {
 	ConfigPaths        []string
+	GithubConfigDir    []string
```

Comment:
This is a list of dirs so GithubConfigDirs

Reply:
done